a08.Evaluator/Evaluator.cs

```diff
@@ -53,16 +83,19 @@ class Evaluator {
             throw new EvalException ($"Unknown token: {token}");
       }
    }
-   readonly Stack<double> mOperands = new ();
-   readonly Stack<TOperator> mOperators = new ();
 
-   void ApplyOperator () {
-      var op = mOperators.Pop ();
-      var f1 = mOperands.Pop ();
-      if (op is TOpFunction func) mOperands.Push (func.Evaluate (f1));
-      else if (op is TOpArithmetic arith) {
-         var f2 = mOperands.Pop ();
-         mOperands.Push (arith.Evaluate (f2, f1));
-      }
+   /// <summary>Resets the evaluator</summary>
+   void Reset () {

```

Comment:
Private methods 'Process ()' & 'Reset ()' should come under '#region Implementation'. Apply this wherever necessary.